### PR TITLE
[docs] remove references to multiobjective optimization being unsupported

### PR DESCRIPTION
**Pull request by odow**
x-ref #3176

### CODE REVIEW

File: docs/src/developers/roadmap.md [hunk 1]
```diff
@@ -25,10 +25,6 @@ represent broad themes that we see as areas in which JuMP could be improved.
    [https://github.com/jump-dev/JuMP.jl/issues/2227](https://github.com/jump-dev/JuMP.jl/issues/2227)
    JuMP has a strong focus on linear, conic and nonlinear optimization problems.
    We want to add better support for constraint programming.
- - Add support for multiobjective problems:
```
Comment (mlubin) [inline docs/src/developers/roadmap.md:28]: Maybe strike-through or add a note that it's done?

Comment (odow) [reply docs/src/developers/roadmap.md:28]: Changed to strike-through

**Review by blegat — APPROVED** [review]

File: docs/src/developers/roadmap.md (hunk 1 above)
Comment (odow) [reply docs/src/developers/roadmap.md:28]: Hmm didn't work: https://jump.dev/JuMP.jl/previews/PR3223/developers/roadmap/. I'll have to find the syntax for documenter

File: docs/src/developers/roadmap.md [hunk 2]
```diff
@@ -25,10 +25,10 @@ represent broad themes that we see as areas in which JuMP could be improved.
...
+ - <s>Add support for multiobjective problems:
```
Comment (odow) [inline docs/src/developers/roadmap.md:28]:
```suggestion
 - ~~Add support for multiobjective problems:
```

File: docs/src/developers/roadmap.md [hunk 3]
```diff
@@ -25,10 +25,10 @@ represent broad themes that we see as areas in which JuMP could be improved.
...
    [https://github.com/jump-dev/JuMP.jl/issues/2099](https://github.com/jump-dev/JuMP.jl/issues/2099)
    JuMP is restricted to problems with scalar-valued objectives. We want to
-   extend this to vector-valued problems.
+   extend this to vector-valued problems.</s> Done [#3176](https://github.com/jump-dev/JuMP.jl/pull/3176)
```
Comment (odow) [inline docs/src/developers/roadmap.md:31]:
```suggestion
   extend this to vector-valued problems.~~ **Done [#3176](https://github.com/jump-dev/JuMP.jl/pull/3176)**
```

**Pull request merged**